mlflow_elasticsearchstore/elasticsearch_store.py

```diff
@@ -35,21 +37,29 @@ def _hit_to_mlflow_experiment(self, hit: Any) -> Experiment:
                           artifact_location=hit.artifact_location,
                           lifecycle_stage=hit.lifecycle_stage)
 
-    def _hit_to_mlflow_run(self, hit: Any) -> Run:
+    def _hit_to_mlflow_run(self, hit: Any, use_latest_metrics: bool = False) -> Run:
         return Run(run_info=self._hit_to_mlflow_run_info(hit),
-                   run_data=self._hit_to_mlflow_run_data(hit))
+                   run_data=self._hit_to_mlflow_run_data(hit, use_latest_metrics))
 
     def _hit_to_mlflow_run_info(self, hit: Any) -> RunInfo:
         return RunInfo(run_uuid=hit.meta.id, run_id=hit.meta.id,
-                       experiment_id=str(hit.experiment_id), user_id=hit.user_id,
-                       status=hit.status, start_time=hit.start_time,
+                       experiment_id=str(hit.experiment_id),
+                       user_id=hit.user_id if hasattr(hit, 'user_id') else None,
+                       status=hit.status,
+                       start_time=hit.start_time if hasattr(hit, 'start_time') else None,
```

Comment:
In what cases there is no user_id or start_time ? Worst case should be that they are set to None in the db no ?